po/es.po

```diff
@@ -16777,147 +16779,152 @@ msgstr "Agregar etiqueta de metadatos de salida"
-#: ../src/libs/geotagging.c:870
+#: ../src/libs/geotagging.c:945
 msgid "open GPX file"
 msgstr "Abrir archivo GPX"
 
-#: ../src/libs/geotagging.c:871
+#: ../src/libs/geotagging.c:946
 msgid "preview"
 msgstr "Vista previa"
 
-#: ../src/libs/geotagging.c:889
+#: ../src/libs/geotagging.c:964
 msgid "GPS data exchange format"
 msgstr "Formato de intercambio de datos GPS"
 
-#: ../src/libs/geotagging.c:1585
-msgid "date time"
+#: ../src/libs/geotagging.c:1661
+msgid "date/time"
 msgstr "fecha/hora"
 
-#: ../src/libs/geotagging.c:1588
+#: ../src/libs/geotagging.c:1664
 msgid ""
-"enter the new date time (yyyy:mm:dd hh:mm:ss)\n"
+"enter the new date/time (yyyy:mm:dd hh:mm:ss)\n"
 "key in the new numbers or scroll over the cell"
 msgstr ""
+"Ingresar la nueva fecha/hora (yyyy:mm:dd hh:mm:ss)\n"
+"tecleando los números o usando la rueda del ratón sobre la celda."
 
-#: ../src/libs/geotagging.c:1594
-#, fuzzy
-msgid "original date time"
-msgstr "imagen original"
+#: ../src/libs/geotagging.c:1670
+msgid "original date/time"
+msgstr "fecha/hora original"
 
-#: ../src/libs/geotagging.c:1600
-#, fuzzy
-msgid "date time offset"
-msgstr "compensación horaria"
+#: ../src/libs/geotagging.c:1676
+msgid "date/time offset"
+msgstr "diferencia horaria"
 
-#: ../src/libs/geotagging.c:1603
+#: ../src/libs/geotagging.c:1679
 msgid "offset or difference ([-]dd hh:mm:ss)"
-msgstr ""
+msgstr "Diferencia horaria ([-]dd hh:mm:ss)"
 
-#: ../src/libs/geotagging.c:1606
-msgid "lock date time offset value to apply it onto another selection"
+#: ../src/libs/geotagging.c:1682
+msgid "lock date/time offset value to apply it onto another selection"
 msgstr ""
 
 #. apply
-#: ../src/libs/geotagging.c:1617
-#, fuzzy
+#: ../src/libs/geotagging.c:1693
 msgid "apply offset"
-msgstr "desplazamiento y"
+msgstr "establecer compensación"
 
-#: ../src/libs/geotagging.c:1617
-#, fuzzy
+#: ../src/libs/geotagging.c:1693
 msgid "apply offset to selected images"
-msgstr "Establecer la compensación horaria en las imágenes seleccionadas"
+msgstr "Establecer la diferencia horaria en las imágenes seleccionadas"
 
-#: ../src/libs/geotagging.c:1622
+#: ../src/libs/geotagging.c:1698
 #, fuzzy
-msgid "apply date time"
+msgid "apply date/time"
 msgstr "fecha/hora"
 
-#: ../src/libs/geotagging.c:1622
+#: ../src/libs/geotagging.c:1698
 #, fuzzy
-msgid "apply the same date time to selected images"
+msgid "apply the same date/time to selected images"
 msgstr "Establecer la compensación horaria en las imágenes seleccionadas"
 
 #. gpx
-#: ../src/libs/geotagging.c:1668
+#: ../src/libs/geotagging.c:1744
 msgid "apply GPX track file..."
 msgstr "adjuntar archivo GPX de recorrido..."
 
-#: ../src/libs/geotagging.c:1669
+#: ../src/libs/geotagging.c:1745
 msgid "parses a GPX file and updates location of selected images"
 msgstr ""
 "Abrir archivo GPX y actualizar la ubicación de las imágenes seleccionadas"
 
-#: ../src/libs/geotagging.c:1681
+#: ../src/libs/geotagging.c:1757
 msgid "gpx file"
 msgstr "archivo gpx"
```

Comment:
Hello @EdgarLux , a minor one gpx is an extension/acronym should be in capital letters (GPX).